spec/rmagick/draw_spec.rb

```diff
@@ -0,0 +1,157 @@
+class Magick::Draw
+  def self._dummy_img_
+    @@_dummy_img_
+  end
+end
+
+describe Magick::Draw do
+  before(:each) { @draw = Magick::Draw.new }
```

Comment:
[I too prefer not to use `let` in general](http://linduxed.com/blog/2014/08/24/writing-more-readable-rspec-tests/), but in this case you've basically done the exact same thing as `let!`, in which case one could use that. Whether memoization is a necessary measure in this case is a different discussion.